### ISSUE TITLE
WalletConnect error

### ISSUE THREAD

**Issue by timofeli** [xmtp/example-chat-react#58]
Hi, I've been working on a few bounties for Shanghai Hackathon and found the following glitch. When I am logging into the app using WalletConnect, I am getting the following error. Have no issues logging in with any other wallet. 

```
Unhandled Runtime Error
Error: PollingBlockTracker - encountered an error while attempting to update latest block:
undefined

Call Stack
PollingBlockTracker._performSync
node_modules/eth-block-tracker/src/polling.js (51:0)
```

Also for MetaMask and WalletConnect, when trying to have a chat between them, I am getting the following error, it says that WalletConnect user is not on xmtp network. 

```
Unhandled Runtime Error
Error: Recipient 0xfC... is not on the XMTP network

Call Stack
Conversations.eval
node_modules/@xmtp/xmtp-js/dist/esm/src/conversations/Conversations.js (79:0)
Generator.next
<anonymous>
fulfilled
node_modules/@xmtp/xmtp-js/dist/esm/src/conversations/Conversations.js (4:42)
```

**Comment by galligan** [comment]
@timofeli with this one:

> Also for MetaMask and WalletConnect, when trying to have a chat between them, I am getting the following error, it says that WalletConnect user is not on xmtp network.

In this case are you using two different browsers? And are you sending messages between two distinct addresses? Or the same account logged in with two different methods?

**Comment by galligan** [comment]
@timofeli just an update on my end—I was just able to sign in with WalletConnect to the example chat app that's deployed at https://xmtp.netlify.app. Can you test to see if you're able to do the same on that URL?

**Comment by timofeli** [comment]
@galligan yes, I was using two different browsers and two distinct addresses. 

Sure, will try it there, thank you.

**Comment by timofeli** [comment]
@galligan had no issues connecting on netlify, maybe it has something to do with me connecting with meta mask before

**Comment by galligan** [comment]
> @galligan had no issues connecting on Netlify, maybe it has something to do with me connecting with meta mask before

I'm wondering if it might be some sort of Infura configuration error then

**Comment by timofeli** [comment]
@galligan I'll check it, thanks again!

**Comment by petermdenton** [comment]
> @galligan I'll check it, thanks again!

Sounds great. Let us know if you continue to experience the issue and we can escalate on our end.

**Comment by haCryptoha** [comment]
> > @galligan I'll check it, thanks again!
> 
> Sounds great. Let us know if you continue to experience the issue and we can escalate on our end.

Hi, I'm working on Shanghai Hackathon and tried to run this chat on my end.
I use MetaMask and Ropsten testnet and connecting wallets works well.
But once I input the account address of another one, it shows
"Recipient is not on the XMTP network" error.
I also try to use https://xmtp.netlify.app/ but it also shows above message.
How can I resolve this? I'm using Google Chrome.

**Comment by petermdenton** [comment]
thanks for submitting this issue. That issue is a temporary (by design) implementation where recipients mist be "known" the network, i.e. have signed in through a client on XMTP.

if you have the target recipient sign in, the error will go away.

**Comment by galligan** [comment]
After some investigation, we uncovered an issue in our testnet environment. The issue has been resolved so this should be working as expected now! @timofeli @haCryptoha

**Comment by neekolas** [comment]
The issue we resolved with the testnet would explain the second issue @timofeli was experiencing, but not necessarily the first. Are you able to confirm the app is working as expected now?

**Comment by timofeli** [comment]
@neekolas yes, it's working as expected, thank you!
